### ISSUE TITLE
Incorrect JSON Output (decimal and scientific)

### ISSUE THREAD

**Issue by jk563** [prashant-ramcharan/courgette-jvm#2]
I'm seeing line values with decimal points and durations with decimal points and scientific notation when using this runner. This contrasts with the numbers in the example documentation (https://relishapp.com/cucumber/cucumber/docs/formatters/json-output-formatter).

For example, I believe that this:
````     
       "line": 11.0,
       ......
          {
            "result": {
              "duration": 1.0702718E7,
              "status": "passed"
            },
           .......
          }
        ],
````

Should look like this:
````     
       "line": 11,
       ......
          {
            "result": {
              "duration": 10702718,
              "status": "passed"
            },
           .......
          }
        ],
````

**Comment by prashant-ramcharan** [comment]
@jk563 good spot. 

Will provide a fix.

**Comment by jk563** [comment]
It's to do with GSON, the line that incorrectly maps it is https://github.com/prashant-ramcharan/courgette-jvm/blob/master/src/main/java/courgette/runtime/CourgetteRunner.java#L166

````
final Object jsonObject = new Gson().fromJson(json, Object.class);
````

**Comment by prashant-ramcharan** [comment]
Thanks - this is now fixed in version 1.3.1.

**Comment by jk563** [comment]
Cheers for the rapid fix, working as expected now!
